pkg/tasks/certs.go

```diff
@@ -158,3 +180,108 @@ func saveCABundleOnControlPlane(s *state.State, _ *kubeoneapi.HostConfig, conn s
 	_, _, err = s.Runner.RunRaw(cmd)
 	return err
 }
+
+func approvePendingCSR(s *state.State, node *kubeoneapi.HostConfig, conn ssh.Connection) error {
+	s.Logger.Infof("Looking for CSRs to approve...")
+
+	// Need to wait for the second CSR to appear
+	time.Sleep(20 * time.Second)
```

Comment:
I think we should leave a log message that KubeOne is waiting 20 seconds for CSRs to appear. We already do that when we wait for other tasks (e.g. for the control plane instance to come up).